tests/std/tests/Dev11_0437519_container_requirements/test.cpp

```diff
@@ -329,42 +329,43 @@ struct emplace_argument {
 
 // clang-format off
 
-#define DEFINE_TYPE(name, dtor, default_ctor, copy_ctor, move_ctor, emplace_ctor, copy_assign, move_assign, emplace_assign) \
-    class name {                                                                                                            \
-    public:                 name(key) { }                                                                                   \
-    private: dtor           ~name() { }                                                                                     \
-    private: default_ctor   name() { }                                                                                      \
-    private: copy_ctor      name(name const&) { }                                                                           \
-    private: move_ctor      name(name&&) { }                                                                                \
-    private: emplace_ctor   name(emplace_argument&&) { }                                                                    \
-    private: emplace_ctor   name(emplace_argument&&, emplace_argument&&) { }                                                \
-    private: emplace_ctor   name(emplace_argument&&, emplace_argument&&, emplace_argument&&) { }                            \
-    private: copy_assign    name& operator=(name const&) { return *this; }                                                  \
-    private: move_assign    name& operator=(name&&) noexcept { return *this; }                                              \
-    private: emplace_assign name& operator=(emplace_argument&&) { return *this; }                                           \
+#define DEFINE_TYPE(name, dtor, def_ctor, copy_ctor, move_ctor, emp_ctor, copy_assign, move_assign, emp_assign) \
+    class name {                                                                                                \
+    public:              name(key) { }                                                                          \
+    private: dtor        ~name() { }                                                                            \
+    private: def_ctor    name() { }                                                                             \
+    private: copy_ctor   name(name const&) { }                                                                  \
+    private: move_ctor   name(name&&) { }                                                                       \
+    private: emp_ctor    name(emp_argument&&) { }                                                               \

```

Comment:
This is failing to compile:

```
1: C:\agent\_work\1\s\tests\std\tests\Dev11_0437519_container_requirements\test.cpp(351): error C2061: syntax error: identifier 'emp_argument'
```

The issue is that `struct emplace_argument` is immediately above, but it hasn't been renamed.

Reply:
July 10th. I had more than month during which I could have built this PR; I can't believe I only ever tested `validate`.